index.html

```diff
@@ -123,10 +123,11 @@ <h1 class="page-header">
            <td>2-3hrs
            <td> <a href="https://github.com/gdisf/teaching-materials/blob/master/jquery2/README.md">Meetup description</a>
         <tr>
-           <td><a href="/jquery-selfpaced/">JS301: jQuery (Online, Self-paced)</a>
+         <td><a href="/jquery-selfpaced/">JS301: jQuery (Online, Self-paced)</a>
            <td>JS200
            <td>2-3hrs
            <td> <a href="https://github.com/gdisf/teaching-materials/blob/master/jquery-selfpaced/README.md">Meetup description</a>
+           <td> <a href="https://github.com/gdisf/teaching-materials/blob/jquery-selfpaced/follow_up.md">Follow Up Email</a>
```

Comment:
This fixes the 2nd part of issue # https://github.com/gdisf/teaching-materials/issues/423 

The 1st part of issue # 423 was merged here: https://github.com/gdisf/teaching-materials/pull/501